tests/headers_test.cc

```diff
@@ -14,30 +14,30 @@ TEST(headers_test, accept)
 
     {
         const auto &media = a1.media();
-        ASSERT_EQ(media.size(), 1U);
```

Comment:
size_t comparisons in ASSERT_EQ should work, even on older googletest.

QQ: Did you only change the tests that absolutely failed to compile on sbuild/i386, or did you change all tests, regardless of their behavior on the incompatible architecture?